src/components/common-components/route-buttons/route-buttons.component.tsx

```diff
@@ -59,27 +60,32 @@ export default function RouteButtonsComponent({ tabs }: { tabs: Tabs[] }): JSX.E
   };
 
   return (
-    <>
-      <div>
-        <Divider />
-        <TabContext value={tab}>
-          <TabList classes={{ indicator: classes.indicatorStyle }} onChange={handleChange}>
-            {tabs.map((tab) => (
-              <StyledTab
-                className={classes.tabStyle}
-                key={tab.label}
-                label={tab.label}
-                value={tab.label}
-              />
-            ))}
-          </TabList>
-          <Divider />
-
+    <div className={"tabs-and-buttons"}>
+      <TabContext value={tab}>
+        <div className={"tabs-row"}>
+          <div className={"tabs-and-buttons"}>
+            <TabList classes={{ indicator: classes.indicatorStyle }} onChange={handleChange}>
+              {tabs.map((tab) => (
+                <StyledTab
+                  className={classes.tabStyle}
+                  key={tab.label}
+                  label={tab.label}
+                  value={tab.label}
+                />
+              ))}
+            </TabList>
+            <Divider />
+          </div>
+          <div className="filler" />
           {tabs.map((tab) => (
-            <TabPanel value={tab.label}>{tab.component}</TabPanel>
+            <TabPanel value={tab.label}>{tab.right_side_buttons}</TabPanel>
```

Comment:
Brakuje `key` przy generowaniu listy.
https://reactjs.org/docs/lists-and-keys.html